snuba/web/query.py

```diff
@@ -63,16 +94,28 @@ def parse_and_run_query(
     """
     # from_clause = request.query.get_from_clause()
     query_metadata = SnubaQueryMetadata(request, get_dataset_name(dataset), timer)
-
+    run_new_query_pipeline_rollout = state.get_float_config(
```

Comment:
here we run old and new in the same deploy and can easily switch using runtime config